libs/deal-ticket/src/components/deal-ticket/deal-ticket.spec.tsx

```diff
@@ -50,7 +50,7 @@ const market: DealTicketMarketFragment = {
       price: '100',
     },
   },
-};
+} as MarketDealTicket;
```

Comment:
suggestion (non blocking): why not just add a type to `market`? IE `const market: MarketDealTicket` that way we can be sure we aren't missing any fields in the mock.

Reply:
Oh presumably this is because lots of fields in the test are not required, so it keeps the test a bit more succinct